cmd/traffic_layout/traffic_layout.cc

```diff
@@ -167,6 +167,7 @@ produce_layout(bool json)
   print_var("remap.config", RecConfigReadConfigPath("proxy.config.url_remap.filename"), json);
   print_var("plugin.config", RecConfigReadConfigPath(nullptr, "plugin.config"), json);
   print_var("ssl_multicert.config", RecConfigReadConfigPath("proxy.config.ssl.server.multicert.filename"), json);
+  print_var("ssl_SNI.config", RecConfigReadConfigPath("proxy.config.ssl.SNI.filename"), json);
```

Comment:
Not hugely favorable to the upper case here, but there are a handful of existing ones (.CA.) already...